From ltsi-dev-bounces@lists.linuxfoundation.org Fri Jul  6 03:28:44 2012
From: Simon Horman <horms@verge.net.au>
Date: Fri,  6 Jul 2012 19:28:31 +0900
Subject: ARM: mach-shmobile: kzm9d: defconfig enable NET_ETHERNET
To: ltsi-dev@lists.linuxfoundation.org
Cc: Nobuhiro Iwamatsu <nobuhiro.iwamatsu.yj@renesas.com>
Message-ID: <1341570511-15470-3-git-send-email-horms@verge.net.au>


Enable NET_ETHERNET which will allow the SMSC911x/921x ethernet driver to
be compiled.

N.B: Not present or necessary upstream
N.B: Already present in LTSI

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 arch/arm/configs/kzm9d_defconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/configs/kzm9d_defconfig
+++ b/arch/arm/configs/kzm9d_defconfig
@@ -52,6 +52,7 @@ CONFIG_IP_PNP_DHCP=y
 CONFIG_UEVENT_HELPER_PATH="/sbin/hotplug"
 # CONFIG_BLK_DEV is not set
 CONFIG_NETDEVICES=y
+CONFIG_NET_ETHERNET=y
 # CONFIG_NET_VENDOR_BROADCOM is not set
 # CONFIG_NET_VENDOR_CHELSIO is not set
 # CONFIG_NET_VENDOR_CIRRUS is not set
